engine/src/main/java/com/core/rpgmaker/RpgMakerStorage.kt

```
@@ -0,0 +1,99 @@
+            ""
+        }
+    }
+
+    @JvmStatic
+    fun write(directory: File?, key: String?, value: String?, extension: String) {
+        try {
+            val file = resolveFile(directory, key, extension) ?: return
+            val bytes = value.orEmpty().toByteArray(StandardCharsets.UTF_8)
+            if (bytes.size > MAX_SAVE_BYTES) return
+            file.outputStream().use { it.write(bytes) }
+        } catch (error: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)